tests/manage/pv_services/test_pvc_evict_ceph_clients.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import pytest
+import random
+import json
+
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework.testlib import tier2
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+class TestPvcEvictCephClients:
+    """
+    Test the ceph-fs PVC (RWX) mount with the following scenarios involving evict ceph-fs clients:
+    1. When two pods tries to mount the same PVC and scheduled on same node
+    2. When two pods tries to mount the same PVC and scheduled on different node
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["interface", "node", "pvc_size"],
+        argvalues=[
+            pytest.param(
+                *[constants.CEPHFILESYSTEM],
+                "same",
+                "500M",
```

Comment:
since the size is the same over the 2 test param sets, do we need the pvc_size as a parameter here?

Reply:
we don't really need that parameter. if I think about it, even interface we dont need as parameter :D
I'll make those changes